Description: Cleanup upstream build.xml:
 - Use local Javadoc index
 - Set test.failonerror to false
 - Remove the -SNAPSHOT suffix from the version
Forwarded: not-needed
Author: Damien Raude-Morvan <drazzib@drazzib.com>
Last-Update: 2013-03-25
--- a/build.xml
+++ b/build.xml
@@ -40,7 +40,7 @@
 
       <property name="name" value="commons-pool"/>
       <property name="title" value="Apache Commons Object Pooling Package"/>
-      <property name="version" value="1.6-SNAPSHOT"/>
+      <property name="version" value="1.6"/>
       <property name="package" value="org.apache.commons.pool.*"/>
 
       <property name="src.dir" value="${basedir}/src"/>
@@ -52,7 +52,7 @@
       <property name="dist.dir" value="${basedir}/dist"/>
       <property name="dist.jar" value="${dist.dir}/${name}-${version}.jar"/>
 
-      <property name="test.failonerror" value="true" /> 
+      <property name="test.failonerror" value="false" /> 
      
       <property name="javadoc.dir" value="${dist.dir}/docs/api"/>
       <property name="javadoc.bottom" value="&lt;small&gt;Copyright &amp;copy; 2001-2012 Apache Software Foundation. Documenation generated ${TODAY}&lt;/small&gt;."/>
@@ -94,7 +94,7 @@
                nonavbar="false"
                serialwarn="false"
                source="${javac.src.version}">
-          <link href="http://java.sun.com/j2se/1.4.2/docs/api"/>
+          <link href="/usr/share/doc/default-jdk-doc/api"/>
       </javadoc>
    </target>
 
